inc/Engine/Common/Clock/ClockInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace WP_Rocket\Engine\Common\Clock;
+
+interface ClockInterface {
```

Comment:
We are over complicate the code here, we can easily use the WP Core function `current_time`

Reply:
My I get your point of view here @Tabrisrp ?

Reply:
This doesn't seem necessary indeed, there is already time management included in WP core